charts/datadog/values.yaml

```diff
@@ -384,6 +384,46 @@ datadog:
     #
     enabled: false
 
+  ## @param securityAgent - object - required
+  ## Enable security agent and provide custom configs
+  #
+  securityAgent:
+    ## @param compliance - object - required
+    compliance:
+      ## @param enabled - boolean - required
+      ## Set this to true to enable compliance checks
+      #
+      enabled: false
+      # @param configMap - string - required - contains compliance benchmarks that will be used
+      #
+      configMap: datadog-compliance-benchmarks
+      # @param checkInterval - duration - required - compliance check run interval
+      #
+      checkInterval: 20m
+
+    ## @param runtime - object - required
+    runtime:
+      ## @param enabled - boolean - optional - default: false
+      ## Set to true to enable the Security Runtime Module.
+      #
+      enabled: false
+
+      ## @param policies - object - required
+      ## Policy files
+      policies:
+        ## @param configMap - string - optional - contains policies that will be loaded
+        ##
+        configMap:
+
+      ## @param syscall_monitor - object - optional
+      ## Syscall monitoring
+      #
+      syscallMonitor:
+        ## @param enabled - boolean - optional - default: false
+        ## Set to true to enable the Syscall monitoring.
+        #
+        enabled: false
```

Comment:
If security-agent does not do `syscallMonitor`, it does nothing right? I think `runtime.enabled` should imply this one, no?

Reply:
I am not sure to be honest, @lebauce @safchain mind clarifying this here?

Reply:
I _think_ it subscribes to audit events too? So not entirely *nothing* 😄